runtime/gc_vlhgc/CollectionSetDelegate.cpp

```diff
@@ -166,7 +166,8 @@ MM_CollectionSetDelegate::createNurseryCollectionSet(MM_EnvironmentVLHGC *env)
 
 			/* For CopyForwardHybrid mode we allow eden regions, which has jniCritical,to be part of nursery collectionSet, those regions would be marked instead of copyforwarded.
 			 * For Non CopyForwardHybrid mode we do not check Eden regions with jniCritical here, because we have already set MarkCompact PGC mode for the case in early. */
-			if (region->getRememberedSetCardList()->isAccurate() && (!isSelectionForCopyForward || !regionHasCriticalRegions || (regionHasCriticalRegions && (_extensions->tarokEnableCopyForwardHybrid || (0 != _extensions->fvtest_forceCopyForwardHybridRatio)) && region->isEden()))) {
+			/* _extensions->tarokEnableCopyForwardHybrid always is true */
```

Comment:
I presume tarokEnableCopyForwardHybrid  will be removed altogether (with a followup OMR change)? if so, referring to it in this comment does not make sense.

Also the comment referring to non-CopyForwardHybrid also should be removed, right?
Even explicitely referring to CopyForwardHybrid (as it will stay as the only mode) is redundant. Those whole comment should collapse to:

"We allow eden regions, which have jniCritical, to be part of nursery collectionSet; those regions would be marked instead of copyforwarded."